discovery-provider/src/exceptions.py

```diff
@@ -17,21 +13,3 @@ class ArgumentError(Base):
 class NotFoundError(Base):
     ''' Invalid arguments passed to request '''
     pass # pylint: disable=W0107
-
-
-def register_exception_handlers(flask_app):
```

Comment:
moved to app.py because this dependency is weird

src.app -> src.exceptions -> src.api_helpers -> src.queries